src/main/java/seedu/address/model/person/NameMatchesKeywordsPredicate.java

```diff
@@ -0,0 +1,42 @@
+package seedu.address.model.person;
+
+import java.util.function.Predicate;
+
+import seedu.address.commons.util.ToStringBuilder;
+
+/**
+ * Tests that a {@code Person}'s {@code Name} strictly matches the keywords given.
+ */
+public class NameMatchesKeywordsPredicate implements Predicate<Person> {
+    private final String keywords;
+
+    public NameMatchesKeywordsPredicate(String keywords) {
+        this.keywords = keywords;
+    }
+
+    @Override
+    public boolean test(Person person) {
+        return keywords.equals(person.getName().toString());
```

Comment:
would it be better if we did something to make it case insensitive? like keywords.toupper and person get name. toupper also? or maybe there are other methods